beacon-chain/blockchain/process_block.go

```diff
@@ -125,7 +125,12 @@ func (s *Service) onBlock(ctx context.Context, signed *ethpb.SignedBeaconBlock,
 		if err != nil {
 			return errors.Wrap(err, "could not fetch finalized state")
 		}
-		s.depositCache.InsertFinalizedDeposits(ctx, int64(finalizedState.Eth1Data().DepositCount-1))
+		eth1Index := int64(finalizedState.Eth1Data().DepositCount - 1)
```

Comment:
Will be nice to include a comment to explain how this eth1 index is derived. 
Reading this is not very clear on how deposit count translates to the index itself.